plugins/gojira-postgresro

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+POSTGRES_RO_PATH=$(dirname ${BASH_SOURCE[0]})/postgresro
+
+#  Add commands to usage
+function gojira-postgresro-commands {
+  cat << EOF
+[postgresro] Commands:
+  up                       gojira's up command is annotated
+
+EOF
+}
+
+function gojira-postgresro-usage {
+  cat << EOF
+
+This plugin adds a read-only replica in addition to the primary postgres database.
+
+EOF
+}
+
+replica_egg() {
+  cat << EOF
+version: '3.5'
+services:
+  # add some PG_RO_HOST and point it to the replica
+  ${GOJIRA_TARGET:-kong}:
+    environment:
+      KONG_PG_RO_HOST: postgresro
+
+  # set replica parameters on the db container
+  db:
+    environment:
+      PG_REP_USER: rep
+      PG_REP_PASSWORD: 123456
+    # overwrite the entrypoint
+    volumes:
+    - ${POSTGRES_RO_PATH}/pg-entrypoint/primary:/docker-entrypoint-initdb.d:rw
+  # Add new readonly replica postgres instance
+  postgresro:
+    image: postgres
+    environment:
+      POSTGRES_HOST_AUTH_METHOD: trust
+      POSTGRES_USER: kong
+      PG_REP_USER: rep
+      PG_REP_PASSWORD: 123456
```

Comment:
(same) Is it needed with `trust` enabled?
```suggestion
```

Reply:
I was thinking it should bypass the authentication with the `trust` method when creating a password-less user. I tried this before and it doesn't work :/

Reply:
Should we make the password overridable?

```suggestion
      PG_REP_PASSWORD: ${KONG_PG_REP_PASSWORD:-123456}
```